### PR TITLE
LPS-110045 FAKE GRADLE CACHE

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:help

**Comment by lawrence-lee** [comment]
ci:help

**Comment by lawrence-lee** [comment]
ci:test:stable

File: modules/sdk/project-templates/project-templates-extensions/src/main/java/com/liferay/project/templates/extensions/util/ProjectTemplatesUtil.java [hunk 1]
```diff
@@ -160,6 +160,12 @@ public static String getArchetypeVersion(String artifactId)
 		return projectTemplateJarVersionsProperties.getProperty(artifactId);
 	}
 
+	public static String getLiferayVersion() {
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-extensions/src/main/java/com/liferay/project/templates/extensions/util/ProjectTemplatesUtil.java]: Hey @lawrence-lee  lets not add this method, as it isn't exactly clear what this should be used for.  Instead in the test case, just locally declare a local variable for liferayVersion.

File: modules/sdk/project-templates/project-templates-form-field/build.gradle [hunk 2]
```diff
@@ -1,6 +1,6 @@
 dependencies {
 	compileOnly group: "com.beust", name: "jcommander", version: "1.72"
-	compileOnly group: "com.liferay", name: "com.liferay.project.templates.extensions", version: "1.0.15"
+	compileOnly group: "com.liferay", name: "com.liferay.project.templates.extensions", version: "1.0.16"
```
Comment (gamerson) [inline modules/sdk/project-templates/project-templates-form-field/build.gradle]: No need to change these dependencies, I think all the needed changes can be localized to the test class.

**Comment by lawrence-lee** [comment]
ci:test:project-templates

**Comment by lawrence-lee** [comment]
ci:test:sf

**Comment by lawrence-lee** [comment]
ci:test:relevant

**Comment by gamerson** [comment]
Pull request submitted to https://github.com/hhuijser/liferay-portal/pull/4049. [See changes here.](https://github.com/gamerson/liferay-portal/compare/e0ec79689a03~1...e0ec79689a03) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**